doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -282,6 +282,13 @@ Heat is the orchestration service using template to provision cloud resources, H
 #### 4.3.1.9 Horizon
 Horizon is the Web User Interface to all OpenStack services. Horizon has services running on the control nodes and no services running on the compute nodes.
 
+#### 4.3.1.10 Cyborg
+Cyborg is the acceleration resources management service. Cyborg depends on Nova and have services running on the control node and compute node:
```

Comment:
The sentence needs to clarify that the cyborg-api, cyborg-conductor and cyborg-db are hosted in the control nodes.
Should there be a mention about the [drivers](https://docs.openstack.org/cyborg/pike/architecture.html) and that cyborg supports only a limited number of [operations](https://docs.openstack.org/cyborg/pike/userdoc/api.html),  BTW the architecture reference should be provided regardless of any content add.

Typo: "... Nova and have services ..." --- "... Nova and has services ..."

Reply:
Also, with Cyborg add we need to add the APIs to RA-1 CH05 -- will open an issue.